web/themes/webspark/renovation/src/sass/bootstrap-asu.scss

```diff
@@ -125,7 +120,6 @@ $theme-colors: map-remove($theme-colors, 1, 3, 5);
 @import "../../node_modules/bootstrap/scss/carousel";
 @import "../../node_modules/bootstrap/scss/spinners";
 @import "../../node_modules/bootstrap/scss/utilities";
-@import "../../node_modules/bootstrap/scss/print";
```

Comment:
Question: why are some bootstrap files removed? Because they no longer exist in the new version?

Reply:
As @davidornelas11  explained, some components are deprecated and were removed in BS5.